assets/javascripts/discourse/widgets/vote-button.js.es6

```diff
@@ -26,33 +26,33 @@ export default createWidget("vote-button", {
     return buttonClass;
   },
 
-  html(attrs) {
-    let buttonTitle;
+  buttonTitle(attrs) {
```

Comment:
kind of a nitpick, but in this case, I prefer to use something like `buildButtonTitle`, `buttonTitle` feels like an instance variable to me, but I'm fine with this, up to you.